addons/snailmail_account/wizard/account_invoice_send.py

```diff
@@ -82,9 +82,12 @@ def send_and_print(self):
     def notify_invalid_addresses(self):
         self.ensure_one()
         self.env['bus.bus'].sendone(
-            (self._cr.dbname, 'res.partner', self.env.user.partner_id.id),
-            {'type': 'snailmail_invalid_address', 'title': _("Invalid Addresses"),
-            'message': _("%s of the selected invoice(s) had an invalid address and were not sent", self.invalid_invoices)}
+            self.env.user.partner_id,
+            'snailmail_invalid_address',
+            {
+                'title': _("Invalid Addresses"),
+                'message': _("%s of the selected invoice(s) had an invalid address and were not sent", self.invalid_invoices),
```

Comment:
payload change: these are useless, they can be built on js
the only useful info to send are the string representation of "invalid_invoices"